mail-filter/postfwd/postfwd-2.00-r1.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
If I do not actually use new EAPI features I want to set the lowest supported EAPI so installing on not-so-uptodate systems is possible

Reply:
Well this going into unstable and the oldest stable portage has EAPI 7, so it doesn't really make sense.

Reply:
By the way, if I commit anything new with EAPI 6 it will receive a comment from some of the guys watching the PRs :)

Reply:
Well, @mschiff is the dedicated maintainer of this package, so you should listen to his wishes in the end. However it's not clear if that is a direct request to lower the EAPI, or just comment on his ways of doing updates? That explanation doesn't make sense to me either, and proxy-maint guide, devmanual and ebuild.skel all say to use latest available EAPI when possible.

If you sync up to receive this ebuild, you should be using the latest stable portage anyway capable of dealing with EAPI-7. I can see someone deliberately not updating their system, but should that be supported through ebuilds? Not in **my** opinion.

Reply:
@juippis I am not completely against bumping EAPI, but normally I do so when using new features from it or the old EAPI is really going to be unsupported soon. SO well, we will leave it bumped then ;)